packages/blockchain-link/package.json

```diff
@@ -53,6 +53,9 @@
     },
     "devDependencies": {
         "@trezor/e2e-utils": "workspace:*",
+        "@trezor/type-utils": "workspace:*",
+        "babel-collect-imports": "^1.1.0",
```

Comment:
what is this good for?

Reply:
I don't know any more than I can read [from the code ](https://github.com/trezor/trezor-suite/blob/chore/no-extraneous-dependencies/packages/blockchain-link/scripts/build.workers.js/#L3).

Reply:
Ah, I see, looks like dead file. There is a mention of `.flow`. I can't find if it is used anywhere.

https://www.npmjs.com/package/babel-collect-imports dead lib you just brought to life

Reply:
ok, let's delete it

Reply:
7ec4114871213958ad4238bbf16e11c5aac99ae5